### DISCUSSION TITLE
Is there any document for Sunbird-Ed release-8.0.0_capacitor setup?

### DISCUSSION THREAD

**Discussion by nidhikachhawa24** [Sunbird-Ed/Community#752] [Documentation]
Hello all,

I'm doing a setup of sunbirdEd **release-8.0.0_capacitor** repository. If anyone has a proper setup document then please provide.

cc: @HarishGangula

**Comment by HarishGangula** [comment]
@nidhikachhawa24  release 8.0.0 is not yet released

**Reply by nidhikachhawa24** [reply]
this branch https://github.com/Sunbird-Ed/SunbirdEd-mobile-app/tree/release-8.0.0_capacitor is not released yet??

@HarishGangula

**Reply by HarishGangula** [reply]
Yes it is not released yet

**Reply by nidhikachhawa24** [reply]
@HarishGangula 

We're currently working on release-5.1 SunbirdEd. But facing issues while updating it to android 14, SDK 34.

As the Google Playstore deadline is approaching for the updation and cordova is not supporting android 14. We're using  **release-8.0.0_capacitor**

But ECML, HTML and youtube contents are getting failed.

Can I know when this release-8.0.0_capacitor will be released?
Also, we have bandwidth so if any development needed, we're happy to contribute.

cc:@baljeetsingh
